services/minespace-web/src/components/common/DocumentTable.tsx

```diff
@@ -205,6 +239,24 @@ export const DocumentTable = ({
     },
   ].filter((action) => allowedTableActions[action.label]);
 
+  const bulkItems: MenuProps["items"] = [
+    {
+      key: "0",
+      icon: <DeleteOutlined />,
```

Comment:
Previously we had used this icon for Delete and <InboxOutlined /> for Archive. Just want to double-check that it didn't get switched by accident.

Reply:
Updated. I also sent a message to clarify same in ux-pod channel.